README.md

```diff
@@ -18,11 +29,126 @@ Current state of the software:
 
 For descriptions of the models supported in our library, discussion of software design, and examples, please refer to the following paper: https://arxiv.org/abs/2205.08144
 
+- Two lightweight interfaces are available for `Python` ([BayesMixPy](https://github.com/bayesmix-dev/bayesmix/tree/master/python#readme)) and `R` (BayesMixR)
+<!-- Add link after merge !!! -->
+
 # Installation
 
+Compiling and building `bayesmix` requires a modern C++ compiler, `git`, the GNU `make` utility, `cmake` (version >= 3.20) and `pkg-config`. Make these packages available varies between operating systems.
+
+**Warning**: Check the version of `cmake` available in your package manager default repository. You may need to install a more recent version in another way. If this is your case, please go to the [official CMake GitHub repository](https://github.com/Kitware/CMake) and follow the installation instructions.
+
+## Requirements - Linux
+
+On Linux, the required packages can be installed via the default package manager. For instance, in Ubuntu it is sufficient to run
+
+```shell
+sudo apt install git
+sudo apt install g++
+sudo apt install make
+sudo apt install cmake
+sudo apt install pkg-config
+```
+
+## Requirements - MacOS
+
+On MacOS, the required packages can be installed via [Homebrew](https://brew.sh/). Once installed, it is sufficient to run
+
+```shell
+brew install git
+brew install g++
+brew install make
+brew install cmake
+brew install pkg-config
+```
+
+## Requirements - Windows
+
+First of all, install `git` via the [Git for Windows](https://gitforwindows.org/) project. Download the [installer](https://github.com/git-for-windows/git/releases/latest) and complete the prompts leaving default choices to install. The Git BASH that comes with this program is the shell we suggest to compile and run `bayesmix`.
+
+On Windows, we also need the installation of a proper C++ toolchain to install the other required packages. `bayesmix` is known complatible with RTools40, RTools42 and RTools43 toolchains. These require slightly different steps to configure, so please follow the appropriate steps below. All toolchains will require updating your `PATH` variable, See [these instructions](https://helpdeskgeek.com/windows-10/add-windows-path-environment-variable/) for details on changing the `PATH` if you are unfamiliar. The following instructions will assume that the default installation directory was used, so be sure to update the paths accordingly if you have chosen a different directory.
```

Comment:
> `bayesmix` is known complatible with RTools40, RTools42 and RTools43 toolchains.

This sentence is unclear and has some spelling mistakes

Reply:
I guess writing docs at night is not the best choice 😂